TensorFlow/Part 2 - Neural Style Transfer/README.md

```diff
@@ -0,0 +1,45 @@
+# Neural Style Transfer - TensorFlow
+
+### Prerequisites
```

Comment:
Same point here, I don't think it makes sense to add the original paper link here, we would be doing the same with pytorch also and I think that is just redundant, we need a smarter way of doing this. 
Also instead of having  ```TensorFlow/Part 2 - Neural Style Transfer/Examples/example 2``` can we just have a ```images``` directory at the top, make the programs refer to this images directory? I'm just trying to think from a reducing redundancy angle.